projects/02-meme-generator-api/implementation/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
+require './app/models/meme'
+
+class ApplicationController < Sinatra::Base
+  configure do
+    set :views, 'app/views'
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  get '/memes' do
+    erb :memes_form
+  end
```

Comment:
Understood last time that the requirements weren't clear enough that we'd go and implement a JSON based API.

Thought some more about it and i suggest you change the implementation so that it accepts a JSON in the request body. This will help you understand the differences between a request having the Content type application/x-www-form-encoded vs application/json and everyone will be working on the same things.

If you need help let me know.